tensorflow/java/src/main/java/org/tensorflow/VariableInput.java

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Let's remove this file for now - since we don't have a Variable class, having this in the code is a bit confusing. Perhaps we can add it when there is a Variable class introduced?